shared/database/source_control/file.py

```diff
@@ -372,44 +377,59 @@ def get_geo_assets(self, session) -> list:
         ).all()
         return assets
 
-    def serialize_geospatial_assets(self, session):
+    def serialize_geospatial_assets(self, session, connection_id = None, bucket_name = None):
         assets_list = self.get_geo_assets(session)
         result = []
         for asset in assets_list:
-            result.append(asset.serialize(session))
+            result.append(asset.serialize(session, connection_id = connection_id, bucket_name = bucket_name))
         return result
 
     def serialize_with_type(self,
                             session = None
                             ):
 
         file = self.serialize_base_file()
-
+        connection_id = self.connection_id
```

Comment:
Just a thought - if in the current context these values will always be self values,
then would it be better to have it as self. later in code?
This is good already. When I first read it though I assumed those values were being pulled from somewhere else. I'm wondering if it would be clearer to have ->

```
project = self.project,
connection_id = self.connection_id,
bucket_name = self.bucket_name)
```

Reply:
Added here: https://github.com/diffgram/diffgram/pull/962/commits/1525779d202783c6a7ee173bd760d8a48a79f380